packages/phone-number-privacy/combiner/test/end-to-end/get-blinded-salt.test.ts

```diff
@@ -102,10 +104,10 @@ describe('Running against a deployed service', () => {
     expect(response.status).toBe(403)
   })
 
-  xdescribe('With funded account', () => {
-    // TODO: Figure out a common way to prefund account to provide quota
+  xdescribe('With enough quota', () => {
```

Comment:
It doesn't work. My understanding is that the combiner e2e tests run against deployed signer services. Is there a way to run them against a local signer service?

Reply:
This will run against whatever endpoint is set for the variable `PHONE_NUM_PRIVACY_COMBINER_SERVICE_URL` in the .env file, which can point to the localhost. Great idea for the `replenishQuota` call! We may want to add a comment to this test case that if it's failing, it may just be that the address needs to be fauceted:
`celotooljs account faucet --account <ADDRESS> --dollar 1 --gold 1 -e <ENV> --verbose`